### DISCUSSION TITLE
정가영 1회차 블로그 피드백 요청 드립니다

### DISCUSSION THREAD

**Discussion by ggaong** [Orm-camp/blog-feedback#9] [동글동글 2기 피드백]
https://gaongg.tistory.com/1

미니 프로젝트에 대한 회고글을 올렸습니다. 피드백 부탁드립니다 :)

**Comment by juuyeoon** [comment]
가영님, 임주연 입니다! 회고글 작성하시느라 고생하셨어요 🙌🏻
귀여우면서도 깔끔한 아이콘들로 글을 정리해 주셔서 내용이 더 쏙쏙 들어오는 기분이었습니다.
다만 카테고리 없음으로 지정되어 있는 것이 가장 처음 눈에 들어왔는데, 블로그를 잘 가꾸고 있다는 느낌을 주기 위해 카테고리도 지정해 보시면 어떨까요?
또 영화 포스터를 누르면 모달창이 나오며 특정 내용만 불러온다고 하셨는데, 어떤 내용들이 불러와지는지도 간략학 적혀있으면 좋겠다는 생각도 들었습니다!
고생 많으셨어요 😊

**Comment by Yu-Kyeong-Seok** [comment]
아이콘을 이용하니 글이 더 눈에 잘들어오는 느낌이 들어 제 블로그 다음글에도 참고해서 작성해봐야겠습니다😊

**Comment by taeyoung0001** [comment]
같이 프로젝트 해서 좋았습니다!
글 쓰신 것 중에서 중간중간에 설명해주는 이미지가 있으면 가독성이 훨씬 좋을것 같네요 ㅎㅎ
늘 화이팅입니다!

**Comment by hyyun1113** [comment]
가영님, 블로그 글 잘 읽었습니다! 딱딱 중요한 내용만 기입해주셔서 읽기 편했고 개발시 이슈사항을 읽으며 당시 얼마나 고심했을지의  상황과, 또 결국 해결했을때의 그 기분까지  잘 느낄수 있는 부분이 참 좋았습니다.
다만 마지막 회고에 위의 이슈를 해결함으로써 어떤부분을 배울 수 있었는지 또는 아쉬움이 있었는지도 추가해서 적으면 내용이 위아래 내용이 연결되어 자연스럽게 읽히지 않을까라는 개인적인 의견을 추가해 봅니다!

**Comment by dayoungh** [comment]
개발 시 이슈사항 부분을 코드로 적으면 뎁스가 한 눈에 보이니까 코드를 활용하신다면 명확하게 눈으로 보여서 더욱 좋을 것 같습니다! 
(이슈사항 쓰지도 않은 제가 할 말은 아니긴 해요...😭)
파트마다 구분도 지어주시고 글 전체가 잘 정리되어 있어 읽기 좋았습니다! 특히 구현기능부분을 짧지만 명확하게 적어주셔서 이해하기에도 쉬웠습니다. 저렇게 딱 할말만 하는게 늘 어려운 저로써는 배우고 싶은 부분입니다ㅎㅎㅎ 잘 읽었습니다👏👏👏👏

**Comment by euneunsol** [comment]
가영님! 좋은 글 잘 읽었습니다!
이슈사항에 대해서 어떤 문제가 있었고 어떻게 해결했다가 다 적혀있어서 이해하기 쉬웠어요! 

또 이모티콘이 적절하게 들어간 것 같아서 읽는데 매우 편했습니다!

**Comment by ehddk** [comment]
가영님 회고록 잘 보았습니다!
키워드 별로 딱딱 적혀있어서 가독성이 좋은 것같아요 .
정말 코딩은 헤매는게 반인것 같지만, 코드 몇줄인데도 그걸 생각해내기까지 수많은 수정과 확인을 하면서 배워가는 것 같아요!
이슈사항까지 적어주셔서 프로젝트에 어떻게 임하셨는지가 잘 느껴지네요 . 같은 팀원이어서 너무 좋았어요😊
